src/Apps/Order2/Hooks/useOrder2Tracking.tsx

```diff
@@ -0,0 +1,76 @@
+import {
+  ActionType,
+  ClickedAskSpecialist,
+  ClickedBuyerProtection,
+  ClickedVisitHelpCenter,
+  ContextModule,
+  OwnerType,
+  PageOwnerType,
+} from "@artsy/cohesion"
+import { useAnalyticsContext } from "System/Hooks/useAnalyticsContext"
+import { Order2HelpLinks_order$data } from "__generated__/Order2HelpLinks_order.graphql"
+import { useMemo } from "react"
+import { useTracking } from "react-tracking"
+
+export const useOrder2Tracking = () => {
+  const { trackEvent } = useTracking()
+  const analytics = useAnalyticsContext()
```

Comment:
non-blocking: So there is a [useCheckoutTracking()](https://github.com/artsy/force/blob/10c99abfe40588b58570ca24aaa4997d7cb7e332/src/Apps/Order2/Routes/Checkout/Hooks/useCheckoutTracking.ts#L23-L40) in the checkout route already. I wonder if this will be confusing later  Would it make sense to have these mixed into that hook? Like if useCheckoutTracking did something like ```
const generalOrderTrackingStuff = useOrder2Tracking()
// ... 
return { ...generalOrderTrackingStuff, submittedOrder, anotherCheckoutSpecificThing }
```